Altis_Life.Altis/stringtable.xml

```diff
@@ -9847,6 +9847,18 @@
             <Portuguese>Você comprou %1 por R$%2</Portuguese>
             <Polish>Kupiłeś %1 za $%2</Polish>
 			<Chinesesimp>你买了一个 %1 花费 $%2</Chinesesimp>
+        </Key>
+		<Key ID="STR_Shop_Veh_Rented">
+            <Original>You rented a %1 for $%2</Original>
+            <Czech>You rented a %1 for $%2</Czech>
+            <Spanish>You rented a %1 for $%2</Spanish>
+            <Russian>You rented a %1 for $%2</Russian>
+            <German>You rented a %1 for $%2</German>
```

Comment:
German Version: "Du hast eine(n) %1 für %2 gemietet"

The other ones should be at least Google-Translated